src/control/server/storage/bdev/backend_json.go

```diff
@@ -237,5 +237,24 @@ func newSpdkConfig(log logging.Logger, req *storage.BdevWriteConfigRequest) (*Sp
 		}
 	}
 
+	if req.HotplugEnabled {
+		for _, ss := range sc.Subsystems {
+			if ss.Name != "bdev" {
+				continue
+			}
+
+			for _, bsc := range ss.Configs {
+				if bsc.Method == SpdkBdevNvmeSetHotplug {
+					bsc.Params = NvmeSetHotplugParams{
+						Enable: true,
+						PeriodUsec: 10 * 1000 * 1000,
+					}
+					break
```

Comment:
Are you sure you want to break out of the loop here and below? This means that only the first matching item will be evaluated. I don't know this code that well so maybe that's what you want.

Reply:
Yes, there is only one hotplug param setting in the SPDK JASON config.

Reply:
Add "var hpFound bool" on line 244, set "hpFound = true" on line 256 and change line 259 to "if hpFound {break}".
This will ensure you check all the subsystems, not just the first. Could also split some code into a helper.